src/lib.rs

```diff
@@ -935,21 +997,58 @@ impl Read for ObjectProperty {
             26 => Self::Pattern(Read::read(input)?),
             27 => Self::PatternTiling(Read::read(input)?),
             28 => Self::PatternOffset(Read::read(input)?),
+            32 => Self::BorderThickness(Read::read(input)?),
```

Comment:
shouldn't this be `Self::Bounce`? I don't think this is right

Reply:
must have tabbed the wrong one that starts with B, yeah should be bounce